tests/unit/-private/properties/triggerable-test.js

```diff
@@ -22,19 +22,16 @@ moduleForProperty('triggerable', function(test) {
     return this.adapter.wait();
   });
 
-  test("calls Ember's triggerEvent helper with event options", function(assert) {
-    assert.expect(3);
+  test("calls Ember's triggerEvent helper with static event options", function(assert) {
+    assert.expect(1);
 
-    let expectedSelector = 'span';
     let page = create({
-      foo: triggerable('keypress', expectedSelector, { eventProperties: { keyCode: 13 } })
+      foo: triggerable('keypress', 'span', { eventProperties: { keyCode: 13 } })
     });
 
     this.adapter.createTemplate(this, page, '<span></span>');
 
     this.adapter.triggerEvent((actualSelector, _, event, options) => {
-      assert.equal(actualSelector, expectedSelector);
-      assert.equal(event, 'keypress');
```

Comment:
These lines can be removed because these expectations are covered in the first test in this file. When I found myself adding two more tests like this one, I thought it would be best to omit redundant assertions to keep the test trim & readable.